shared/common-adapters/web-view.native.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import {isIOS} from '../constants/platform'
+import {WebView} from 'react-native'
+// $FlowFixMe
+import WKWebView from 'react-native-wkwebview-reborn'
+
+const wv = isIOS ? WKWebView : WebView
```

Comment:
will this work? does the import on android work? you likely just want to make a web-view.ios.js and web-view.android.js file

Reply:
Yes it has empty imports for Android. I tried .ios and android but couldn't make flow work. I'll tweak more and see what's going on ...

Reply:
ok if the import doesn't crash we can leave it

Reply:
@songgao I think you need to replace `.native.js` with `.{ios,android}.js` to get it to recognize those.

Reply:
@jzila I think that's what I did; will try again.